### PR TITLE
[Docker check] fixes: cgroup parsing + docker.mem.in_use computation

### PR DESCRIPTION
**Pull request by jiquiame**
### What does this PR do?
Fixes:
- Fix docker memory metrics parsing from cgroup files
- Fix docker.mem.in_use metric computation

### Motivation

Support case where some of the memory metrics where missing

### CODE REVIEW

File: releasenotes/notes/[Docker-check]-fixes:-cgroup-parsing-+-docker.mem.in_use-computation-e390c8f2c2541d2f.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
```
Comment (xvello) [inline releasenotes/notes/[Docker-check]-fixes:-cgroup-parsing-+-docker.mem.in_use-computation-e390c8f2c2541d2f.yaml]: Could you please make two items, like this:
```
fixes:
  - Fix docker memory limit metrics parsing from cgroup files
  - Fix docker.mem.in_use metric computation
```

**Review by xvello — CHANGES_REQUESTED** [review]
👍 for the fixes, just a minor change on the reno

File: releasenotes/notes/[Docker-check]-fixes:-cgroup-parsing-+-docker.mem.in_use-computation-e390c8f2c2541d2f.yaml (hunk 1 above)
Comment (antoinepouille) [reply releasenotes/notes/[Docker-check]-fixes:-cgroup-parsing-+-docker.mem.in_use-computation-e390c8f2c2541d2f.yaml]: 👍

**Review by xvello — APPROVED** [review]
LGTM

**Pull request merged**